.ci/build_images.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+images=("php" "fpm" "apache-php")
+tagPrefix=("" "php-" "php-")
+latestTag="7.1"
+
+scriptFailed=1
+inspectFailed=0
+
+cwd=$(pwd)
+for (( i=0; i<${#images[@]}; i++ )); do
```

Comment:
A more portable and readable way would be to use the following syntax:
`for IMAGE in $IMAGES; do`